src/App/Tracker/Controller/AbstractHookController.php

```diff
@@ -419,6 +419,12 @@ protected function processStatus($action, $currentStatusId = null)
 			case 'closed':
 				$status = 10;
 
+				// If the action is closed and this is a pull request, check if the request was merged and set the status to "Fixed in Code Base"
+				if ($this->type == 'pulls' && $this->hookData->pull_request->base->merged)
```

Comment:
This should be `$this->hookData->pull_request->merged` ;) Just spent 30 minutes on testing and gigging why status is always `Closed`